### PR TITLE
squad || DrQA || pytorch 0.4.1 + k80/v100 ec2, 1070ti entries

### PR DESCRIPTION
**Pull request by brettkoonce**
guide: https://github.com/hitvoice/DrQA/blob/master/dawnbench.md

### CODE REVIEW

File: SQuAD/train/dawn_drqa_1k80-ec2.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "v1.0",
+    "author": "Runqi Yang, Facebook ParlAI, Brett Koonce",
+    "authorEmail": "runqiyang@gmail.com",
+    "framework": "Pytorch 0.4.1",
+    "model": "DrQA",
+    "hardware": "1 K80",
+    "costPerHour": 0.900,
```
Comment (codyaustun) [inline SQuAD/train/dawn_drqa_1k80-ec2.json:8]: Where is this `costPerHour` coming from? What cloud service did you use?

File: SQuAD/train/dawn_drqa_1p100-ec2.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "v1.0",
+    "author": "Runqi Yang, Facebook ParlAI, Brett Koonce",
+    "authorEmail": "runqiyang@gmail.com",
+    "framework": "Pytorch 0.4.1",
+    "model": "DrQA",
+    "hardware": "1 V100",
+    "costPerHour": 3.06,
```
Comment (codyaustun) [inline SQuAD/train/dawn_drqa_1p100-ec2.json:8]: Where is this `costPerHour` coming from? What cloud service did you use?

File: SQuAD/train/dawn_drqa_1k80-ec2.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "timestamp": "2018-09-29",
+    "codeURL": "https://github.com/hitvoice/DrQA",
+    "commitHash": "d9081fea48de31711c17a9af6c2b1b99dadcfd71",
+    "misc": {
```
Comment (codyaustun) [inline SQuAD/train/dawn_drqa_1k80-ec2.json:12]: For convenience, can you add hyperparameters like batch size, optimizer, learning rate schedule, and dropout rate to `misc`?

File: SQuAD/train/dawn_drqa_1nv1070ti-dev.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    "author": "Runqi Yang, Facebook ParlAI, Brett Koonce",
+    "authorEmail": "runqiyang@gmail.com",
+    "framework": "Pytorch 0.4.1",
+    "model": "DrQA",
+    "hardware": "1 NVidia 1070 Ti (dev box)",
+    "timestamp": "2018-09-30",
+    "codeURL": "https://github.com/hitvoice/DrQA",
+    "commitHash": "d9081fea48de31711c17a9af6c2b1b99dadcfd71",
+    "misc": {
+      "paper_link": "http://www-cs.stanford.edu/people/danqi/papers/acl2017.pdf"
```
Comment (codyaustun) [inline SQuAD/train/dawn_drqa_1nv1070ti-dev.json]: For convenience, can you add hyperparameters like batch size, optimizer, learning rate schedule, and dropout rate to `misc`?

File: SQuAD/train/dawn_drqa_1p100-ec2.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    "timestamp": "2018-09-29",
+    "codeURL": "https://github.com/hitvoice/DrQA",
+    "commitHash": "d9081fea48de31711c17a9af6c2b1b99dadcfd71",
+    "misc": {
+      "paper_link": "http://www-cs.stanford.edu/people/danqi/papers/acl2017.pdf"
```
Comment (codyaustun) [inline SQuAD/train/dawn_drqa_1p100-ec2.json]: For convenience, can you add hyperparameters like batch size, optimizer, learning rate schedule, and dropout rate to `misc`?

**Review by codyaustun — CHANGES_REQUESTED** [review]
Looks good overall. I just need a few more details to verify costPerHour

File: SQuAD/train/dawn_drqa_1k80-ec2.json (hunk 1 above)
Comment (brettkoonce) [reply SQuAD/train/dawn_drqa_1k80-ec2.json:8]: p2 on ec2, I can tag it better!

File: SQuAD/train/dawn_drqa_1p100-ec2.json (hunk 2 above)
Comment (brettkoonce) [reply SQuAD/train/dawn_drqa_1p100-ec2.json:8]: p3 on ec2, will tag it better!

File: SQuAD/train/dawn_drqa_1k80-ec2.json (hunk 3 above)
Comment (brettkoonce) [reply SQuAD/train/dawn_drqa_1k80-ec2.json:12]: Going to double-check all of these, let me chat w/ Runqi!

Comment (brettkoonce) [reply SQuAD/train/dawn_drqa_1k80-ec2.json:12]: @codyaustun updated, let me know if you need anything else!

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
Thanks! Looks good to me

**Comment by brettkoonce** [comment]
Thanks @codyaustun!

**Comment by hitvoice** [comment]
@codyaustun A crucial question: which version of the SQuAD dataset is used for the question answering leaderboard now, 1.0 or 2.0?  The hyperlink [https://rajpurkar.github.io/SQuAD-explorer/](https://rajpurkar.github.io/SQuAD-explorer/) on the page led to 1.0 before but leads to 2.0 now and I haven't found any explicit instructions. The SQuAD 2.0 dataset is much more difficult and the results will vary a lot.

**Comment by codyaustun** [comment]
@hitvoice for the purposes of rolling submissions, we will continue to use the SQuAD 1.0 dataset. If we introduce a new version of the benchmark, we would update the tasks to the most relevant dataset at the time.

**Comment by hitvoice** [comment]
@codyaustun Got it. Thanks for the explanation!
